apps/services/auth/public-api/src/app/modules/delegations/meDelegations.controller.ts

```diff
@@ -82,8 +82,13 @@ export class MeDelegationsController {
     },
   })
   @Audit<DelegationDTO[]>({
-    resources: (delegations) =>
-      delegations.map((delegation) => delegation?.id ?? ''),
+    resources: (delegations) => {
+      const filteredIds = delegations
+        .map((delegation) => delegation.id ?? undefined)
+        .filter((id): id is string => Boolean(id))
```

Comment:
I was bit confused because the autocomplete imported from class-validator. So took a look at snaer's PR and found the util. I included it here because it is amazingly handy for this use case.